pages/01.HDM Documentation/04.get-started/10.vcd/default.md

```diff
@@ -445,9 +444,7 @@ Navigate to the _Licensing_ page on the HDM vCenter Plugin
    <td>
 <ul>
 
-<li> Set the WAN subnet for both prem & cloud.
-<li> Set the gateway for WAN both sides.
-<li> DON'T Skip this step.
+<li> since we are using single network, this steps can be skipped.
```

Comment:
@sampat on cloud there are two networks, HDM_internal and HDM_WAN, you need to set this correctly.